Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1106,8 +1106,8 @@ Vector<LayoutRect> RenderGrid::gridItemsLayoutRects()
 {
     Vector<LayoutRect> items;
 
-    for (RenderBox* gridItem = currentGrid().orderIterator().first(); gridItem; gridItem = currentGrid().orderIterator().next())
-        items.append(gridItem->frameRect());
+    for (RenderBox& gridItem : childrenOfType<RenderBox>(*this))
+        items.append(gridItem.frameRect());
```

Comment:
We don't care about order anymore?

Reply:
good catch, I'm revering this specific change so it maintains the order. however, the function of this logic isn't referenced anywhere.